buildpack-run.sh

```diff
@@ -1 +1,6 @@
+#!/usr/bin/env bash
+
 git clone git@github.com:MTES-MCT/ecobalyse-private.git
+
+# Scalingo should set the $SOURCE_VERSION variable
+./bin/checkout-ecobalyse-private-branch.sh $SOURCE_VERSION
```

Comment:
What calls this bash script? I couldn't find any docs about it the Scalingo one

I really wish we could use anything else than bash and sed for advanced scripting and parsing here :/